codebase/commons/src/main/resources/application.conf

```diff
@@ -1 +1,2 @@
-cassandra.contactPoint = "127.0.0.1"
\ No newline at end of file
+cassandra.contactPoint = "127.0.0.1"
+cassandra.contactPoint = ${?CASSANDRA_CONTACT_POINT}
```

Comment:
I quess only `cassandra.contactPoint = ${?CASSANDRA_CONTACT_POINT} ` is needed

Reply:
Ok then 👍